seed/java-sdk/alias/src/main/java/com/seed/alias/core/ClientOptions.java

```diff
@@ -93,11 +99,17 @@ public Builder addHeader(String key, Supplier<String> value) {
             return this;
         }
 
+        public Builder timeout(int timeout) {
```

Comment:
nit: there should be a javadoc above this method that says `Override the timeout in seconds. Defaults to 60 seconds.`

Reply:
Done.